config/bravo_sdf.cfg

```diff
@@ -39,3 +39,15 @@ lcio_bin_jar = /sdf/home/b/bravo/.m2/repository/org/lcsim/lcio/2.7.4-SNAPSHOT/lc
 [EvioToLcio]
 hps_java_bin_jar = /sdf/home/b/bravo/.m2/repository/org/hps/hps-distribution/4.5-SNAPSHOT/hps-distribution-4.5-SNAPSHOT-bin.jar
 conditions_url = jdbc:sqlite:/sdf/group/hps/users/bravo/db/hps_conditions_test.db
+
+[PEDE]
+#command = /full/path/not/in/PATH/pede
+param_map = /sdf/group/hps/users/bravo/hps-mille/align/data/param_maps/hpsSvtParamMap_2019.txt
+pede_minimization = /sdf/group/hps/users/bravo/hps-mille/align/data/pede_steer/steer_minimization_template.txt
+
```

Comment:
Not sure if there is a good "standard" path for where the parameter maps and/or pede templates are. I just made this path up to show that it could be anywhere in your HPS user space basically.